assets/src/apps/scheduler/ScheduleEditor.tsx

```diff
@@ -110,6 +137,22 @@ export const ScheduleEditor: React.FC<SchedulerProps> = ({
     'Cancel',
   );
 
+  const { Modal: unsavedModal, showModal: showUnsavedModal } = useBackdropModal(
+    <div>
+      <p>
+        If you view without saving, you’ll see the last saved changes—not your most recent edits.
```

Comment:
Is this the correct message?  The message seems to imply that I have a choice here:  that if I view without saving that I will see my last saved changes, not my most recent edits - but the implementation here forces a save?   Maybe the message should read something more along the lines of "Previewing your schedule requires that you save all changes first".

Reply:
Urgh. I copy pasted from figma. Will update the text